src/components/Buttons/BadgeIconButton.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const BadgeIconButton = (props) => {
+  const { badgeNumber, icon, badgeIconButtonWrapper, onClick } = props;
+  return (
+    // <div>
+      <button onClick={onClick} className={`btn ${badgeIconButtonWrapper}`}>
```

Comment:
suggestion: if possible, can give a more descriptive function name instead on onClick (like onBadgeClick)

Reply:
Yes, that would make code more readable. thanks for pointing this out.